apps/doc-gen/src/app/docgen/utils/normalizeContractPath.ts

```diff
@@ -0,0 +1,17 @@
+
+export function normalizeContractPath(contractPath: string, returnPath?: boolean): string | string[]{
```

Comment:
what do you want to do here?

Reply:
was catering for a situation where full file path is not desired.